src/strategies/eth-with-balance/index.ts

```diff
@@ -0,0 +1,29 @@
+import { strategy as ethBalanceOfStrategy } from '../eth-balance';
+import { verifyResults } from '../ocean-marketplace/oceanUtils';
+
+export const author = 'Aron van Ammers';
```

Comment:
This should be your GitHub username
```suggestion
export const author = 'AronVanAmmers';
```